app/routes/api/claim.js

```diff
@@ -218,6 +223,38 @@ const addHerdToPreviousClaims = async (herdClaimData, applicationReference, crea
   ))
 }
 
+const emitHerdMIEvents = async ({ sbi, herdData, tempHerdId, herdGotUpdated }) => {
+  const { herdId, herdVersion, herdName, species: herdSpecies, cph: herdCph, herdReasons } = herdData
+
+  await raiseHerdEvent({ sbi, message: 'Herd associated with claim', type: 'claim-herdAssociated', data: { herdId, herdVersion } })
```

Comment:
I think we need to include the claim reference here, otherwise we can;t indicate what the herd was associated with

Reply:
those arent in the ADR, is that ok?

Reply:
Yes, they are in the wording, but don't get mentioned in the table - basically says we will also include relevant existing fields to the entries (and that the take will only describe new columns introduced). It's pretty much only claim reference, and agreement here. The other on submission ones are wholly her focused so don't need the claim ref

Reply:
ok cool ill add those!